org.pitivi.Pitivi.json

```diff
@@ -0,0 +1,294 @@
+{
+    "app-id": "org.pitivi.Pitivi",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "3.22",
+    "finish-args": ["--command=pitivi",
+                    "--share=ipc",
+                    "--socket=x11",
+                    "--socket=session-bus",
+                    "--socket=pulseaudio",
+                    "--socket=wayland",
+                    "--talk-name=ca.desrt.dconf",
+                    "--talk-name=org.freedesktop.Notifications",
+                    "--filesystem=host"],
+    "sdk": "org.gnome.Sdk",
+    "rename-appdata-file": "pitivi.appdata.xml",
+    "rename-desktop-file": "pitivi.desktop",
+    "rename-icon": "pitivi",
+    "copy-icon": true,
+    "build-options": {
+        "cflags": "-O0 -g",
+        "cxxflags": "-O0 -g",
+        "env": {
+          "PYTHON": "python3",
+          "GST_PLUGIN_SYSTEM_PATH": "/app/lib/gstreamer-1.0/"
+        }
+    },
+    "modules": [
+        {
+            "name": "yasm",
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "http://www.tortall.net/projects/yasm/releases/yasm-1.3.0.tar.gz",
+                    "sha256": "3dce6601b495f5b3d45b59f7d2492a340ee7e84b5beca17e48f862502bd5603f"
+                }
+            ]
+        },
+        {
+            "name": "sound-theme-freedesktop",
+            "sources": [
+                {
+                    "type": "git",
+                    "url": "https://anongit.freedesktop.org/git/sound-theme-freedesktop.git",
+                    "branch": "0.8"
+                }
+            ]
+        },
+        {
+            "name": "gsound",
+            "sources": [
+                {
+                    "type": "git",
+                    "url": "https://git.gnome.org/browse/gsound",
+                    "branch": "1.0.2"
+                }
+            ]
+        },
+        {
+            "name": "x264",
+            "config-opts": ["--enable-shared", "--enable-static", "--enable-pic", "--disable-lavf"],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "http://download.videolan.org/pub/x264/snapshots/x264-snapshot-20140212-2245-stable.tar.bz2",
+                    "sha256": "5d98e9e4faf6dd55e7193ed379aff477b8acbda6777758956ef7e5f05067be18"
+                }
+            ]
+        },
+        {
+            "name": "numpy",
+            "build-options": {
+              "build-args": ["--share=network"],
```

Comment:
Flathub is sandboxed, you cannot grant network access. You have to fix their build system to work offline.